src/App.jsx

```diff
@@ -1,33 +1,61 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function getNormalizedTExt(text, query) {
+  const normalizedText = text.toLowerCase().includes(query);
+
+  return normalizedText;
+}
+
+const getFilterFilms = (movies, query) => {
+  let preparedMovies = [...movies];
+  let newQuery = query;
+
+  if (newQuery) {
+    newQuery = newQuery.trim().toLowerCase();
+    preparedMovies = preparedMovies.filter(movie => (getNormalizedTExt(
+      movie.description, newQuery,
+    ) || getNormalizedTExt(movie.title, newQuery)
+    ));
+  }
+
+  return preparedMovies;
+};
```

Comment:
let's optimize your code 
```suggestion
function getNormalizedTExt(text, query) {
  return text.toLowerCase().includes(
    query.trim().toLowerCase()
  );
}

const getFilterFilms = (movies, query) => {
  if (query) {
    return preparedMovies.filter(movie =>
      getNormalizedTExt(movie.description, query) || getNormalizedTExt(movie.title, query)
    );
  }
  return movies;
};
```